clients/pkg/promtail/targets/docker/tagetmanager_test.go

```diff
@@ -0,0 +1,111 @@
+package docker
```

Comment:
This file is called `tagetmanager_test.go`. That should probably be `targetmanager_test.go`.